src/components/Site/Site.jsx

```diff
@@ -137,6 +127,14 @@ class Site extends React.Component {
    * @return {array}       - ...
    */
   _strip = array => {
+    let anchorTitleIndex = array.findIndex(item => item.name === 'index.md');
```

Comment:
This is the main change, I am finding the object which name is `index.md` and copy it at the beginning of the array, then proceed to remove it from its original position.

This is the result:
![image](https://user-images.githubusercontent.com/1002461/41790413-d2c550f0-7652-11e8-8370-b497f4280393.png)